apps/builder/app/builder/features/style-panel/shared/model.tsx

```diff
@@ -376,6 +377,17 @@ export const useParentComputedStyleDecl = (property: StyleProperty) => {
   return useStore($store);
 };
 
+export const getInstanceStyleDecl = (
+  property: StyleProperty,
+  instanceSelector: InstanceSelector
+) => {
+  return getComputedStyleDecl({
+    model: $model.get(),
+    instanceSelector,
+    property,
+  });
```

Comment:
very implicit model access, somebody will use getInstanceStyleDecl and get stale result. Just import model where necessaey.

Reply:
How it would be stale, as model is read exactly here.

Reply:
if use in render values will not subscribe to model
I really don't see a value in this function

Reply:
no value just to hide model ;)

Reply:
$model is not exported. I have zero need to subscribe